### DISCUSSION TITLE
Question about streaming audio

### DISCUSSION THREAD

**Discussion by gpeles** [geckosio/geckos.io#88] [Help / Questions] [answered]
Hi, thanks for making this package!
I hope my question will make sense - I'm developing a multiplayer game that uses players' audio input to control the game. As audio plays an important role in the game, the players should hear one another while playing. In earlier prototypes I used Socket.IO for messaging and WebRTC for streaming audio. If I use this package for messaging, will I have to establish new RTC connections for streaming audio or could I use the connections already established by the package?

**Comment by c4b4d4** [comment]
Hey @gpeles did you ever manage to run audio-streaming over Geckos.io? I'm interested in this feature.

**Comment by yandeu** [comment, marked as answer]
Geckos.io is not developped for using audio. I recommend you using a client-only webrtc audio library for this.
